modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -498,39 +442,9 @@ private void createTableLocally(long causalityToken, String name, UUID tblId, in
         InternalTableImpl internalTable = new InternalTableImpl(name, tblId, new Int2ObjectOpenHashMap<>(partitions),
                 partitions, netAddrResolver, clusterNodeResolver, txManager, tableStorage);
 
-        var schemaRegistry = new SchemaRegistryImpl(v -> {
-            if (!busyLock.enterBusy()) {
-                throw new IgniteException(new NodeStoppingException());
-            }
-
-            try {
-                return tableSchema(tblId, v);
-            } finally {
-                busyLock.leaveBusy();
-            }
-        }, () -> {
-            if (!busyLock.enterBusy()) {
-                throw new IgniteException(new NodeStoppingException());
-            }
-
-            try {
-                return latestSchemaVersion(tblId);
-            } finally {
-                busyLock.leaveBusy();
-            }
-        });
-
-        var table = new TableImpl(internalTable, schemaRegistry);
-
-        tablesVv.update(causalityToken, previous -> {
-            var val = new HashMap<>(previous);
-
-            val.put(name, table);
+        var table = new TableImpl(internalTable, null);
```

Comment:
It is the main place in source where we are creating a table, hence it looks weird that we use a constructor with null value.